services/shhext/chat/encryption_multi_device_test.go

```diff
@@ -12,110 +14,93 @@ func TestEncryptionServiceMultiDeviceSuite(t *testing.T) {
 	suite.Run(t, new(EncryptionServiceMultiDeviceSuite))
 }
 
+type serviceAndKey struct {
+	e []*EncryptionService
```

Comment:
i love to use this shortcuts myself, but hard to understand what is e when you read code for first times